cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/wizard-basic/management-credential-step/ManagementCredentials.tsx

```diff
@@ -68,10 +70,18 @@ function ManagementCredentials(props: Partial<StepProps>) {
     }, []);
 
     useEffect(() => {
+        const fetchKeyPairs = async () => {
+            try {
+                setKeyPairLoading(true);
+                const keyPairs = await AwsService.getAwsKeyPairs();
+                setKeyPairs(keyPairs);
+                setKeyPairLoading(false);
+            } catch (e) {
```

Comment:
How about putting the `setKeyPairLoading(false)` into a `finally` block and doing something with the error (at least console, but maybe display?)

Reply:
fixed.